README.md

```diff
@@ -95,6 +95,8 @@ Use the "Table of Contents" menu on the top-left corner to explore the list.
 - [vite-ts-vue2-todo](https://github.com/nabaonan/todos-action/tree/master/vue2-vite-ts) - Vue2 + ts + Vue-Router3 + Pinia1 + Composition API.
 - [vite-vue2-typescript](https://github.com/byoungd/vite-vue2-typescript) - Vue2 TypeScript starter template using composition-api.
 - [vite-vue2-mpa-template](https://github.com/selene-y/vite-vue2-mpa-template) - A multi-page application template using Vue2.
+- [vite-vue2-ts-starter](https://github.com/logue/vite-vue2-ts-starter) - Vue2 TypeScript startar template usng vue-property-decorator, vue-router, vuex.
+  - [vite-vue2-ts-vuetify-starter](https://github.com/logue/vite-vue2-vuetify-ts-starter) - Vuetify added to the above template. Includes basic template.
```

Comment:
incorrect list level

Reply:
It's a derivative of vite-vue2-ts-starter and has changed levels in the sense that it's a UI framework with Vuetify and its optimized settings.